tools/run_tests/run_xds_tests.py

```diff
@@ -2541,62 +2541,96 @@ def create_global_forwarding_rule(gcp,
 
 
 def get_health_check(gcp, health_check_name):
-    result = gcp.compute.healthChecks().get(
-        project=gcp.project, healthCheck=health_check_name).execute()
-    gcp.health_check = GcpResource(health_check_name, result['selfLink'])
+    try:
+        result = gcp.compute.healthChecks().get(
+            project=gcp.project, healthCheck=health_check_name).execute()
+        gcp.health_check = GcpResource(health_check_name, result['selfLink'])
+    except Exception as e:
+        gcp.errors.append(e)
+        gcp.health_check = GcpResource(health_check_name, None)
 
 
 def get_health_check_firewall_rule(gcp, firewall_name):
-    result = gcp.compute.firewalls().get(project=gcp.project,
-                                         firewall=firewall_name).execute()
-    gcp.health_check_firewall_rule = GcpResource(firewall_name,
-                                                 result['selfLink'])
+    try:
+        result = gcp.compute.firewalls().get(project=gcp.project,
+                                             firewall=firewall_name).execute()
+        gcp.health_check_firewall_rule = GcpResource(firewall_name,
+                                                     result['selfLink'])
```

Comment:
Recreating the firewall was a nice feature, are we going to remove it?

Reply:
Is it nice? Once the project is enrolled in our internal firewall enforcer, my impression was the rule would be deleted before most tests would even have the chance to complete. Since it's unstable in that environment, I'd rather just explicitly require the user to create the firewall rule themselves (since they will, typically, have to recreate it after deletion anyways)

Reply:
The firewall do get deleted overtime. The recreating feature saves me from running a firewall-creating-busy-loop on the side. The internal GCEE request will take sometime.

Reply:
I can add this back in. But, in my recent experience, the automatic deletion is now frequent enough that long-running tests (many of these test cases take >10 minutes) will almost certainly fail due to the firewall rule being deleted mid-run. So this only really helps with short test cases like ping pong, and seemed likely to lead to confusion when manually running the longer test cases.

Reply:
Make sense.